website/docs/guides/eks-getting-started.html.md

```diff
@@ -0,0 +1,594 @@
+  name = "terraform-eks-demo-cluster"
+
+  assume_role_policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "eks.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+POLICY
+}
+
+resource "aws_iam_role_policy_attachment" "demo-cluster-AmazonEKSClusterPolicy" {
+  policy_arn = "arn:aws:iam::aws:policy/AmazonEKSClusterPolicy"
+  role       = "${aws_iam_role.demo-cluster.name}"
+}
+
+resource "aws_iam_role_policy_attachment" "demo-cluster-AmazonEKSServicePolicy" {
+  policy_arn = "arn:aws:iam::aws:policy/AmazonEKSServicePolicy"
+  role       = "${aws_iam_role.demo-cluster.name}"
+}
+```
+
+#### EKS Master Cluster Security Group
+
+This security group controls networking access to the Kubernetes masters.
+We will later configure this with an ingress rule to allow traffic from the
+worker nodes.
+
+```hcl
+resource "aws_security_group" "demo-cluster" {
+  name        = "terraform-eks-demo-cluster"
+  description = "Cluster communication with worker nodes"
+  vpc_id      = "${aws_vpc.demo.id}"
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags {
+    Name = "terraform-eks-demo"
+  }
+}
+
+# OPTIONAL: Allow inbound traffic from your local workstation external IP
+#           to the Kubernetes. You will need to replace A.B.C.D below with
+#           your real IP. Services like icanhazip.com can help you find this.
+resource "aws_security_group_rule" "demo-cluster-ingress-workstation-https" {
+  cidr_blocks       = ["A.B.C.D/32"]
+  description       = "Allow workstation to communicate with the cluster API Server"
+  from_port         = 443
+  protocol          = "tcp"
+  security_group_id = "${aws_security_group.demo-cluster.id}"
+  to_port           = 443
+  type              = "ingress"
+}
+```
+
+#### EKS Master Cluster
+
+This resource is the actual Kubernetes master cluster. It can take a few minutes to
+provision in AWS.
+
+```hcl
+resource "aws_eks_cluster" "demo" {
+  name            = "${var.cluster-name}"
+  role_arn        = "${aws_iam_role.demo-cluster.arn}"
+
+  vpc_config {
+    security_group_ids = ["${aws_security_group.demo-cluster.id}"]
+    subnet_ids         = ["${aws_subnet.demo.*.id}"]
+  }
+
+  depends_on = [
+    "aws_iam_role_policy_attachment.demo-cluster-AmazonEKSClusterPolicy",
+    "aws_iam_role_policy_attachment.demo-cluster-AmazonEKSServicePolicy",
+  ]
+}
+```
+
+#### Obtaining kubectl Configuration From Terraform
+
+If you are planning on using `kubectl` to manage the Kubernetes cluster, now
+might be a great time to configure your client. The below Terraform output
+generates a sample `kubectl` configuration to connect to your cluster.
+
+You can verify cluster access via `kubectl version` displaying server version
+information in addition to local client version information.
+
+```hcl
+locals {
+  kubeconfig = <<KUBECONFIG
+
+
```

Comment:
Are the two empty lines desired here?

Reply:
When I was developing the original demo configuration, the extra lines were helpful for distinguishing the "config" output from the "TF" output in the terminal. I don't think they'll hurt anything being in here still.